### PR TITLE
Unit test framework

### PR DESCRIPTION
**Pull request by olevs**
Added test that covers qSlicerLiverMarkupsModule, vtkSlicerLiverMarkupsLogic, and all LiverMarkups MRML data nodes.

### CODE REVIEW

**Comment by olevs** [comment]
@RafaelPalomar: This pull request is now ready to merge, even if there are more tests that needs to be written. This also contain a fix for CircleCI, so it would be good to merge it soon.

File: LiverResections/qSlicerLiverResectionsModule.cxx [hunk 1]
```diff
@@ -126,7 +126,7 @@ QStringList qSlicerLiverResectionsModule::categories() const
 //-----------------------------------------------------------------------------
 QStringList qSlicerLiverResectionsModule::dependencies() const
 {
-  return QStringList() << "LiverMarkups";
+  return QStringList() << "LiverMarkups" << "Markups";
```
Comment (RafaelPalomar) [inline LiverResections/qSlicerLiverResectionsModule.cxx:129]: `LiverMarkups` already depends on `Markups` this looks redundant to me. Is there any reason to add it here explicitly @olevs, @gatangen ?

**Review by RafaelPalomar — CHANGES_REQUESTED** [review]

File: LiverResections/qSlicerLiverResectionsModule.cxx (hunk 1 above)
Comment (olevs) [reply LiverResections/qSlicerLiverResectionsModule.cxx:129]: The problem was that the python test failed without this addition. We couldn't get the dependencies loaded any other way

Comment (RafaelPalomar) [reply LiverResections/qSlicerLiverResectionsModule.cxx:129]: I see. Let's keep it then.

**Pull request merged**